src/background.js

```diff
@@ -12,3 +12,21 @@ chrome.runtime.onInstalled.addListener(function () {
     ]);
   });
 });
+
+// keep a list of connected content scripts so that we can tell them to toggle mute
+const ports = new Set();
+// listen to global keyboard handler events
+chrome.commands.onCommand.addListener(function (command) {
+  if (command === "toggle-meeting-mute" && ports.size > 0) {
```

Comment:
nit: I'd invert this and return early. I.e. `if (command !== 'toggle' || ports.size === 0) { return; }` (but using newlines)